Apply Title heading class to Events page #2081

lindseyindev opened the pull request:
Fixes #2081

### What changes did you make and why did you make them ?

  -Added appropriate Title classes to events page headings per [figma design file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8552%3A0)

  -Updated heading elements to be more semantically correct for screen readers because the styling is now through the Title class.
- Added color property to title-meetup class in /_events.scss set to #333333 per figma design file because there was a universal h4 color set to pink that was being applied to the title5 elements on the meetup events column
  

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>
  
![screencapture-hackforla-org-events-2022-01-25-19_51_30](https://user-images.githubusercontent.com/89806097/151742579-51ef1225-bab7-4938-9727-4867380a82bd.png)


</details>

<details>
<summary>Visuals after changes are applied</summary>
  

![screencapture-localhost-4000-events-2022-01-30-21_16_24](https://user-images.githubusercontent.com/89806097/151742538-b33f085f-c8c5-42cd-993f-ad735a586cff.png)


</details>

Commit:
added Title class to specified  event headings

Georgema20 commented:
Everything looks good, but when I pull it up, there is a visual change in the left column as a result of the class change put in onboarding and those titles. Please make sure to reflect this properly in the visuals after change section with screenshots. Also, please put in a before screenshot as well. 

Everything else looks pretty good. Keep it up! Make the changes then I can approve PR @lindseyindev

lindseyindev commented:
@Georgema20 thank you for letting me know, I updated the PR comment for your review!

Georgema20 commented:
Nice work!

SAUMILDHANKAR commented:
Review ETA: EOD 1/5/22
Availability: 6 hrs

SAUMILDHANKAR reviewed (dismissed):
@lindseyindev Hi Lindsey, thanks for taking up this issue. I have been comparing your events page with that at Figma and I see that the titles onboarding, workshop, happy hour should be dark gray whereas in your branch they don't appear so. As per the design in Figma, they should be dark gray.

Commit:
added title-meetup text-color class to match figma

lindseyindev commented:
I updated the scss for the title-meetup class which applies to just those titles that were showing as pink from styling on the h4 element, I updated the color to #33333 per the figma file and I also updated the after picture in my submission!

SAUMILDHANKAR commented:
Revised review ETA: EOD 1/8
Availability: Saturday 2-5 PM
Couldn't finish the review as per this estimate

SAUMILDHANKAR commented:
Revised review ETA: EOD 1/10
Availability: Monday 2-4 PM

SAUMILDHANKAR commented on _includes/events-page/header-container-content.html:
```diff
@@ -1,7 +1,7 @@
   <div class="show-video">
     <div class="header-text-margin--events">
-      <h1>Events</h1>
+      <h1 class="Title1">Events</h1>
```
Hi @lindseyindev, please make all title lower case, example: title1, title2 and so on

SAUMILDHANKAR reviewed (commented):
title1, title2 etc in lower case.

SAUMILDHANKAR commented:
> I updated the scss for the title-meetup class which applies to just those titles that were showing as pink from styling on the h4 element, I updated the color to #33333 per the figma file and I also updated the after picture in my submission!

Great job in catching the color of h4. As per design system on Figma, text color should be black or white and red is reserved for links, so just wondering if we should remove red color from h4. This can be part of future issue, we can confirm with design in sunday meeting.

Commit:
lowercased title classes

lindseyindev commented:
Updated the title classes to be lower case & submitting for re-review!

Would definitely recommend a suggestion for design & for other new developers, like how we mentioned in the meeting differences between design and codebase, that the figma class should be text-cased to match what will be coded especially if that's the reference for devs 
or it's noted in the issue if for some reason figma classes can't be lowercased

JessicaLucindaCheng commented:
- Availability: 6 hours
- Review ETA: Tue, Jan 18, 2022

JessicaLucindaCheng commented on _includes/events-page/right-col-content.html:
```diff
@@ -22,7 +22,7 @@ <h2 class="event-title">
         {% for day in days-of-week %}
         <div id="{{day}}">
-            <h3 class="day-header-1">{{day}}</h3>
+            <h5 class="day-header-1 title7">{{day}}</h5>
```
Line 25 _includes/events-page/right-col-content.html should be an `h4` html header tag, not `h5`.

JessicaLucindaCheng commented on _sass/components/_events.scss:
```diff
@@ -348,6 +348,7 @@
   text-align: left;
   font-weight: 700;
   margin-bottom: 16px;
+  color: #333333;
```
Instead of adding the line `color: #333333`, could you apply the class `color-black` to the header html tags that need it?

For your reference, the `color-black` class comes from here: https://github.com/hackforla/website/blob/508dc989cf45a8b339fb430e87e22f751bc17db1/_sass/elements/_color-styles.scss

JessicaLucindaCheng reviewed (changes requested):
@lindseyindev Thanks for working on this issue for so long! In addition to my inline comments, could you update your before and after images with a full screen shot of the page?
 
You can use this extension: https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?hl=en

JessicaLucindaCheng commented:
@lindseyindev Please provide your availability and ETA for working on the requested changes above.

lindseyindev commented:
> @lindseyindev Please provide your availability and ETA for working on the requested changes above.

I did not see this until now, I should have this updated within 24 hours!!!

Commit:
utilizing color-black and updated h5 to h4

JessicaLucindaCheng commented:
- Availability: 6 hours
- Review ETA: Thursday, January 27, 2022

JessicaLucindaCheng commented:
@lindseyindev Sorry, I don't have time to review it today. So, I will be delaying my review by one day.
- Availability: 6 hours
- ETA: Friday, January 28, 2022

JessicaLucindaCheng commented on _includes/events-page/right-col-content.html:
```diff
@@ -22,7 +22,7 @@ <h3 class="event-title title4">
         {% for day in days-of-week %}
         <div id="{{day}}">
-            <h5 class="day-header-1 title7">{{day}}</h5>
+            <h4 class="day-header-1 title7">{{day}}</h4>
```
Add the class `color-black` to this header so that the color of the font for days of the week headers match the current website.

The SCSS class `.color-black` is defined in [line 50 in _sass/elements/_color-styles.scss](https://github.com/hackforla/website/blob/713e9504e59ae8c4f3a1575e118f49376a40ec15/_sass/elements/_color-styles.scss#L50) as `.color-black { color: $color-black; }`

JessicaLucindaCheng reviewed (changes requested):
@lindseyindev Thanks for making the requested changes! I think we are almost there with this pull request. Once you make changes based on my in-line request for a change, can you update your before and after images in your [original post/comment](https://github.com/hackforla/website/pull/2645#issue-1090867092)? The before should look like the current webpage. Thank you for all your hard work!

Commit:
added text color property to day-header-1

lindseyindev commented:
I didn't realize changing the h5 tag to h4 undid the previous fix I did to make the color adhere to the figma so I fixed it by targetting the day heading class.

Also somehow a pull request I reviewed is being added as a commit when I pushed my code even though that was on a separate branch and I never committed it

The code with the <strong> tags in the commit history.. https://github.com/hackforla/website/pull/2718 not sure how to undo that on my end -- I dont see a way to remove that part of the commit

JessicaLucindaCheng commented:
- Availability: 3 hours
- Review ETA: Thu, Feb 3, 2022

SAUMILDHANKAR commented:
Revised review ETA: 2/3/22
Availability: 6-8 PM Thursday

JessicaLucindaCheng reviewed (changes requested):
@lindseyindev 
> The code with the ** tags in the commit history.. #2718 not sure how to undo that on my end -- I dont see a way to remove that part of the commit**

I'm wondering if doing the following would fix the problem you are having:
i. Update your forked gh-pages so that it is in sync with the hackforla/website gh-pages. Here are instructions for [how to update your forked gh-pages](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27e-working-on-an-issue-5-incorporating-changes-from-upstream).
ii. Then merge your gh-pages into your topic branch locally.
iii. Then push the merged changes in your topic branch locally to your remote topic branch in your forked repo.

Commit:
Merge branch 'gh-pages' into apply-heading-classes-2081

lindseyindev commented:
I think it worked but I'm not entirely sure how.. like how pushing to my remote fork also pushes to this PR at the same time

JessicaLucindaCheng commented:
- Availability: 6 hours
- Re-review ETA: Mon, Feb 7, 2022

JessicaLucindaCheng reviewed (approved):
@lindseyindev Thanks for updating your topic branch!
> I think it worked but I'm not entirely sure how.. like how pushing to my remote fork also pushes to this PR at the same time

To answer your question, your pull request is connected to your topic branch:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/31293603/152871921-0cc036c2-e6d4-4fe5-bff9-b2ab3287da70.png">

Thus, any changes to your topic branch will show up in your pr.
